src/app.js

```diff
@@ -196,7 +196,9 @@ export function app(props, container) {
   }
 
   function patchElement(parent, element, oldNode, node, isSVG, nextSibling) {
-    if (oldNode == null) {
+    if (oldNode === node) {
```

Comment:
@andyrj If this is all we need, we can simplify this down a bit more as follows.

```js
if (oldNode===node){
    } else 
```

Reply:
Good call I wasn't taking the return at bottom into account for bytes.

Reply:
Okay, please rebase. 🙏